File: tests/fixtures/python/Fields.py
from __future__ import annotations


class Fields:
    total: int = 0

    def __init__(self, count: int) -> None:
        self.name: str = "anonymous"
        self.enabled: bool = False
        self.count = count

    def get_name(self) -> str:
        return self.name

File: tests/fixtures/java/Fields.java
package com.example;

public class Fields {

    private static int total = 0;
    private String name = "anonymous";
    private int count;
    private boolean enabled;

    public Fields(int count) {
        this.count = count;
    }

    public String getName() {
        return name;
    }
}
